gateway/src/node/client_handling/websocket/connection_handler/fresh.rs

```diff
@@ -130,14 +128,18 @@ pub(crate) struct FreshHandler<R, S> {
     pub(crate) shutdown: ShutdownToken,
 
     // currently unused (but populated)
```

Comment:
Is this still up to date? Asking because the same client side comment was removed